packages/core/src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -100,7 +100,7 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                 return await this._executeWithVaulted(payment, instrumentId, shouldSetAsDefaultInstrument);
             }
 
-            const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}?method=${methodId}`));
+            const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}`, {params: {method: methodId}}));
```

Comment:
@BC-AOrtiz As discussed with Jose, could you review stripe strategy when you have time? Thanks